NuGet.Config

```diff
@@ -6,4 +6,16 @@
     <add key="roslyn" value="https://dotnet.myget.org/F/roslyn-tools/api/v3/index.json" />
     <add key="azure-sdk-tools" value="https://azuresdkartifacts.blob.core.windows.net/azure-sdk-tools/index.json" />
   </packageSources>
-</configuration>
\ No newline at end of file
+  <config>
+		<add key="globalPackagesFolder" value="restoredPackages" />
```

Comment:
Is this really needed?

Reply:
this is especially helpful when sdks are generated as part of the automated flow and things needs to cleaned and start clean.
So yes this is needed.

Reply:
This has impact on others working in this repo as it means we essentially cannot depend on the nuget package caching feature which adds time and overhead. Is the reason you want to scope these is because there are conflicting packages with the same version?